project-base/storefront/tailwind.config.js

```diff
@@ -92,19 +92,6 @@ module.exports = {
             actionInvertedBorderDisabled: '#A3ACBD',
             actionInvertedBorderHovered: '#7892BC',
 
-            secondaryText: '#FFFFFF',
-            secondaryTextActive: '#FFFFFF',
-            secondaryTextDisabled: '#B6C3D8',
-            secondaryTextHovered: '#004EB6',
-            secondaryBackground: '#004EB6',
-            secondaryBackgroundActive: '#004EB6',
-            secondaryBackgroundDisabled: '#FAFAFA',
-            secondaryBackgroundHovered: '#FFFFFF',
-            secondaryBorder: '#004EB6',
-            secondaryBorderActive: '#004EB6',
-            secondaryBorderDisabled: '#B6C3D8',
-            secondaryBorderHovered: '#004EB6',
```

Comment:
Don't we want to keep it here even though we don't use them as a lead for future projects how to use the colour system?